bootstrap/reset_db.py

```
@@ -0,0 +1,65 @@
+        print("You can now run 'bootstrap.cmd' to recreate tables and reseed data.")
+    except Exception as e:
+        print(f"Error dropping database: {e}")
+        raise
+    finally:
+        await engine.dispose()
+
+if __name__ == "__main__":
+    _confirm_reset()
+    try:
+        asyncio.run(drop_all())
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)